app/views/layouts/application.html.erb

```diff
@@ -34,8 +35,8 @@
                 </div>
               <% end %>
               <% if current_medwallet_account != nil %>
-                <%= button_to "Sign-Out", destroy_medwallet_account_session_path, :class => "btn btn-outline-secondary" %>
-                <%= button_to "User Home", { :action => 'profile', :id => current_medwallet_account.id }, :class => "btn btn-outline-secondary" %>
+                <%= button_to "Sign-Out", destroy_medwallet_account_session_path, :class => "btn btn-outline-warning", method: :delete %>
+                <%= button_to "User Home", { :action => 'profile', :id => current_medwallet_account.id }, :class => "btn btn-outline-success", method: :get %>
```

Comment:
TIL [`button_to` has a default `method: :post`.](https://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to) Probably because usually you use `link_to` instead if you want to make a GET request. Then you could style the link as a button using the bootstrap classes you're already using.